packages/eslint-config/flat-config.js

```diff
@@ -162,7 +162,21 @@ const flatConfig = [
         }
       ],
       'import/export': 'error',
-      'import/order': 'error',
+      'import/order': [
```

Comment:
Having a consistent order makes sense to me, I do not really have an opinion of which order is preferrable, out of curiosity, did you come up with the list yourself or is it some standard you follow here?

Reply:
this is the standard documented in the rule itself.